src/FlaUI.WebDriver.UITests/ElementTests.cs

```diff
@@ -210,5 +210,18 @@ public void ActiveElement_Default_IsSupported()
 
             Assert.That(activeElement.Text, Is.EqualTo("Invoked!"));
         }
+
+        [TestCase(["ClassName", "TextBox"])]
```

Comment:
Shouldn't we add a test case for a non-existing attribute that it returns `null`?

Reply:
Done.